### ISSUE TITLE
CocoaHeads OKC assets folder

### ISSUE THREAD

**Issue by kacollins** [techlahoma/user-groups#1251]
Need a folder for CocoaHeads OKC:
https://github.com/techlahoma/user-groups/tree/master/sponsored

**Comment by billygoat** [comment]
Is there a document somewhere that lists what should be in that folder? I browsed user-group-docs and didn't see anything.

**Comment by kacollins** [comment]
You're right, we need to add that. For now, can you copy from one from another group? One of the main things we need is the group's logo.

**Comment by billygoat** [comment]
will do, thanks

**Comment by billygoat** [comment]
Thanks!


[Domino’s](https://www.dropbox.com/s/owiav2bzyg5jw73/IMG_0991.jpeg?dl=1)

**Comment by billygoat** [comment]
Thanks!


[Church’s Chicken ](https://www.dropbox.com/s/ookqyzv92e18zur/PNG%20image.png?dl=1)

**Comment by billygoat** [comment]
Thanks!


[Domino’s Pizza](https://www.dropbox.com/s/imex4nitt0w76r8/Camera%202019-05-18%20at%206.08.15%20PM.jpeg?dl=1)
